php-practice_1.php

```diff
@@ -1,41 +1,173 @@
 <?php
 // Q1 変数と文字列
 
+ $name = '「江藤」';
+
+$newname = '私の名前は' . $name . 'です。';
+echo  '私の名前は' . $name . 'です。';
 
 // Q2 四則演算
 
+$num= 5*4;
```

Comment:
@saotsuna 
可読性をあげるために半角スペース開けましょう！
他にも類似箇所あれば修正お願いします。
```suggestion
$num = 5 * 4;
```

Reply:
確認済み